NSS-Day-Cohort-49/tabloid-mvc-tabloid-mvc-cassettes
# Subscribe sx

#### Changes Made
1. Added file Subscription Model, Subscription and ISubscription Repository.

1. Added subscribe and unsubscribe functionality.

#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout subscribe-sx
    ```
2. Open a new Terminal tab (?T) and navigate to the server directory.
3. Test app functionality.
    > Make sure subscribe and unsubscribe works and view that all posts by author you are subscribed to shows unsubscribe
    > Review all other views work. 
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
